doc/concept/unique-proofs-using-tries-merkleproofs.md

```diff
@@ -0,0 +1,109 @@
+# Unique proofs using tries / merkle proofs
+
+A unique digital proof is prerequisite for claims and immutability of certificates
+
+## Patlet
+
+Through a session of workshops we have a theoritical approach to an implementation that gives us the ability to have unique claims using proofs / subproofs for a batch of data transactions per registry.
+
+## Problem
+
+The problem being that digital copies cannot be distinguished from their original counterparts. Using a cryptographic method we should be able to proof uniqueness of a claim to mitigate double spending, issuance and claims. The solution will be  generic and subsequently applied to the Energy Track and Trace problem space.
+
+## Story
+
+In order to facilitate making claims of energy a digital asset with unique claims at scale we are faced with a set of challenges that must be addressed in order to achieve a production ready, scalable solution.
+
+We have conducted research in tries / merkle tree structures in order to be able to locally batch a set of transactions involving issuance of production and consumption certificates and the subsequent claims on these certficates. 
+
+In order to facilitate loose coupling between the processes of validation each registry will have to process a trie per batch of transactions and publish them to prove that the underlying data has noot been changed. The trie has a root hash that will change by any change in the underlying dataset - not only is the trie a verification of the dataset, it can also contain subproofs that consumers can present validating their claims without disclosing information about the consumption, value, origin etc, but as a purely verifiable cryptographic proof to third-parties.
```

Comment:
```suggestion
To facilitate loose coupling between the processes of validation, each registry will have to process a trie per batch of transactions and then publish them, to prove that the underlying data has not been changed. 
The trie has a root hash that will change by any change in the underlying dataset, and hence the trie is not only a verification of the dataset, it can also contain subproofs that consumers can present validating their claims without disclosing information about the consumption, value, origin etc, but as a purely verifiable cryptographic proof to third-parties.